src/main/scala/org/apache/spark/sql/delta/commands/VacuumCommand.scala

```diff
@@ -119,12 +119,11 @@ object VacuumCommand extends VacuumCommandImpl {
       val hadoopConf = spark.sparkContext.broadcast(
         new SerializableConfiguration(sessionHadoopConf))
       val basePath = fs.makeQualified(path).toString
-      var isBloomFiltered = false
```

Comment:
Yeah this is probably an artifact of our code management. This is where we integrate with our internal bloom filter support. Can you please keep this as a var.

Reply:
Ok, changed back.